fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx

```diff
@@ -0,0 +1,165 @@
+npm install -g fern-api
+```
+
+<Callout intent='info'>
+The above CLI command is a global command that you can run from any location. The `fern` commands in the following steps must be run from within your repository.
+</Callout>
+
+### Step 5: Generate your documentation
+
+Run the following command:
+
+```shell
+fern generate --docs
+```
+
+When prompted, log into and connect your GitHub account.
+
+Once the documentation is generated, Fern displays the URL where you can view the published documentation. For example:
+
+```shell
+┌─
+│ ✓  petstore-ferndef.docs.buildwithfern.com
+└─
+```
+
+Instead of `petstore-ferndef`, you should see the organization name that you added to `docs.yml` in Step 3. 
+
+### Step 6: Customize your documentation
+
+You must run `fern generate --docs` after any modifications to re-generate and publish your documentation site.
+
+To preview updates to your documentation before publishing changes, run `fern generate --docs --preview`.
+
+To modify the API Reference:
+
+- Update or replace the Fern Definition files in the `definition/` folder. See our documentation on [how to define your API with Fern Definition](define-your-api/ferndef/overview).
+
+To modify the other docs pages or add new pages:
+
+- Update the files located in the `docs/pages/` folder, such as `welcome.mdx`. See our documentation on [how to write content with Markdown and MDX](/fern-docs/content/write-markdown).
+
+To configure site navigation, styles, and see other configuration options:
+
+- See [Fern Docs Configuration Overview](/fern-docs/config/overview).
+
+To learn about Fern's built-in component library you can use in Markdown:
+
+- See the [Component Library](/fern-docs/content/component-library/).
+
+### Step 7: Set up a custom domain
+
+If you wish to use a custom subdomain like `https://docs.YOUR_ORGANIZATION.COM` or a subpath like `https://YOUR_ORGANIZATION.COM/docs`, you can subscribe to the [Starter plan](https://buildwithfern.com/pricing). Once subscribed, update `docs.yml` with the custom domain configuration, replacing YOUR_ORGANIZATION with your own:
+
+<CodeBlocks>
+    <CodeBlock title='Custom subdomain'>
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com
+      custom-domain: docs.YOUR_ORGANIZATION.COM
+    ```
+    </CodeBlock>
+    <CodeBlock title='Custom subpath'>
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com
+      custom-domain: YOUR_ORGANIZATION.COM/docs
+    ```
+    </CodeBlock>
+</CodeBlocks>
+
+Good luck creating beautiful and functional documentation! 🌿
+
+---
+
+## Support
+
+Need help? Email us at [support@buildwithfern.com](mailto:support@buildwithfern.com) or join our [Discord community](https://discord.com/invite/JkkXumPzcG).
+
+## Customer showcase
+
+Your docs can look this good:
+
+- [Flatfile's API Reference](https://reference.flatfile.com/api-reference/events/create-an-event)
+- [Sugeragent's Docs](https://docs.superagent.sh/)
+- [Credal's Docs](https://docs.credal.ai/)

```

Comment:
Can you swap this to:

1. Flagright https://docs.flagright.com/
2. Superagent
3. Flatfile